src/main/java/edu/cornell/kfs/fp/document/service/impl/CULegacyTravelServiceImpl.java

```diff
@@ -193,41 +85,13 @@ public String getTravelUrl() {
 	public void setTravelUrl(String travelUrl) {
 		this.travelUrl = travelUrl;
 	}
-
-	
-	/**
-	 * @param client
-	 */
-	private void configureWebServiceClient(Client client) {
-		Endpoint endpoint = client.getConduitSelector().getEndpoint();
-	    endpoint.getEndpointInfo().setAddress(updateTripEndpoint);
-	    client.getConduitSelector().setEndpoint(endpoint);
-
-	    HTTPConduit httpConduit = (HTTPConduit) client.getConduit();
-	    HTTPClientPolicy httpClientPolicy = new HTTPClientPolicy();
-	    httpClientPolicy.setConnectionTimeout(36000);
-	    httpClientPolicy.setAllowChunking(false);
-	    httpConduit.setClient(httpClientPolicy);	
-	}
 	
 	/**
      * Determines if the passed in CULegacyTravelIntegrationInterface (Disbursement Voucher Document, or Distribution of Incum Document
      * is associated with a trip.  Either way, this function sets internval of document for the trip association status code and the trip ID.
      */
     public boolean isCULegacyTravelIntegrationInterfaceAssociatedWithTrip(CULegacyTravelIntegrationInterface cuLegacyTravelIntegrationInterace) {
-        if (StringUtils.isBlank(cuLegacyTravelIntegrationInterace.getTripAssociationStatusCode())) {
-            String tripId = getLegacyTripID(cuLegacyTravelIntegrationInterace.getDocumentNumber());
-            if (StringUtils.isBlank(tripId)) {
-                cuLegacyTravelIntegrationInterace.setTripAssociationStatusCode(TRIP_ASSOCIATIONS.IS_NOT_TRIP_DOC);
-                cuLegacyTravelIntegrationInterace.setTripId(StringUtils.EMPTY);
-            } else {
-                cuLegacyTravelIntegrationInterace.setTripAssociationStatusCode(TRIP_ASSOCIATIONS.IS_TRIP_DOC);
-                cuLegacyTravelIntegrationInterace.setTripId(tripId);
-            }
-        }
-        boolean retVal = StringUtils.equals(TRIP_ASSOCIATIONS.IS_TRIP_DOC, 
-                cuLegacyTravelIntegrationInterace.getTripAssociationStatusCode());
-        return retVal;
```

Comment:
If it's not practical to remove the last few methods of the CULegacyTravelService, then I would suggest keeping this boolean check in place for compatibility with older legacy-travel-related docs (unless the functionals don't care about seeing the related travel links on old DVs or DIs), and also create a follow-up user story to look into removing the last few legacy travel remnants from KFS.